diff --git a/test-vfs/dbstatus.test b/test-vfs/dbstatus.test
index 8d5d834..de81a44 100644
--- a/test-vfs/dbstatus.test
+++ b/test-vfs/dbstatus.test
@@ -397,20 +397,20 @@ ifcapable shared_cache {
       CREATE TABLE t1(a, b, c);
       INSERT INTO t1 VALUES(1, 2, 3);
     }
-    do_cacheused_test 4.0.1 db { 4568 4568 }
+    do_cacheused_test 4.0.1 db { 5112 5112 }
     do_execsql_test 4.1 {
       CREATE TEMP TABLE tt(a, b, c);
       INSERT INTO tt VALUES(1, 2, 3);
     }
-    do_cacheused_test 4.1.1 db { 9000 9000 }
+    do_cacheused_test 4.1.1 db { 10112 10112 }
   
     sqlite3 db2 file:test.db?cache=shared
-    do_cacheused_test 4.2.1 db2 { 4568 2284 }
-    do_cacheused_test 4.2.2 db { 9000 6716 }
+    do_cacheused_test 4.2.1 db2 { 5112 2556 }
+    do_cacheused_test 4.2.2 db { 10112 7556 }
     db close
-    do_cacheused_test 4.2.3 db2 { 4568 4568 }
+    do_cacheused_test 4.2.3 db2 { 5112 5112 }
     sqlite3 db file:test.db?cache=shared
-    do_cacheused_test 4.2.4 db2 { 4568 2284 }
+    do_cacheused_test 4.2.4 db2 { 5112 2556 }
     db2 close
   }
 }
